ci/travis/test-wheels.sh

```diff
@@ -102,12 +102,12 @@ elif [[ "$platform" == "macosx" ]]; then
 
     # Run a simple test script to make sure that the wheel works.
     INSTALLED_RAY_DIRECTORY=$(dirname "$($PYTHON_EXE -u -c "import ray; print(ray.__file__)" | tail -n1)")
-    $PYTHON_EXE -m pytest -v "$INSTALLED_RAY_DIRECTORY/$TEST_SCRIPT"
+    # $PYTHON_EXE -m pytest -v "$INSTALLED_RAY_DIRECTORY/$TEST_SCRIPT"
 
     if (( $(echo "$PY_MM >= 3.0" | bc) )); then
       # Run the UI test to make sure that the packaged UI works.
-      $PIP_CMD install -q aiohttp google grpcio psutil requests setproctitle
-      $PYTHON_EXE -m pytest -v "$INSTALLED_RAY_DIRECTORY/$UI_TEST_SCRIPT"
+      # $PIP_CMD install -q aiohttp google grpcio psutil requests setproctitle
+      # $PYTHON_EXE -m pytest -v "$INSTALLED_RAY_DIRECTORY/$UI_TEST_SCRIPT"
```

Comment:
@edoakes This [caused an error](https://travis-ci.com/ray-project/ray/jobs/260426067#L247) :( every statement inside the `if` is commented out, which makes it an empty block, which is a syntax error. If you need a no-op, just putting a colon as the statement.